src/resources/Pipelines/ResultRankings/ResultRankings.ts

```diff
@@ -73,4 +76,12 @@ export default class ResultRankings extends Resource {
             {method: 'post', body: resultRanking, headers: {'Content-Type': 'application/json'}}
         );
     }
+
+    duplicate(pipelineId: string, resultRankingsId: string) {
+        return this.api.post<ResultRanking>(
+            this.buildPath(ResultRankings.getResultRankingsDuplicateUrl(pipelineId, resultRankingsId), {
```

Comment:
Is the function `getResultRankingsDuplicateUrl` reusable? If not you can pass directly ``${ResultRankings.getBaseUrl(pipelineId)}/duplicate/${resultRankingsId}`` to `this.buildPath`, it's not necessary to extract it if we only use it once for duplicate call.

Reply:
Nope it was for the purpose of keeping the URLs for different endpoints close together

Reply:
I removed it but then realized the unit test I added uses that function so it would be better to keep that URL in one place so it isn't duplicated between the source and unit test